staging/script.jellyfin-fixer/resources/lib/player.py

```
@@ -0,0 +1,95 @@
+            params = {
+                id_name: dbid,
+                "properties": ["resume"],
+            }
+
+            json_dict['params'] = params
+            query = json.dumps(json_dict)
+            result = send_kodi_json(f'Get resume point for type {library_type} with dbid: {dbid}', query)['result']
+            if result:
+                try:
+                    resume_point = result[result_key]['resume']['position']
+                except:
```

Ruff (0.13.2)

Do not use bare `except`

(E722)